### PR TITLE
Relax node minor version

### PR DESCRIPTION
**Pull request by chrisalley**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "src"
   ],
   "engines": {
-    "node": ">=18.16"
+    "node": ">=18"
```
Comment (tagliala) [inline package.json:29]: Doesn't this include also 19, 20?

Comment (chrisalley) [reply package.json:29]: It will. This change will allow the package to be added to projects using Node versions between 18.0 and 18.16.

Comment (tagliala) [reply package.json:29]: Ok thanks, yes, got it now

**Review by tagliala — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (chrisalley) [reply package.json:29]: That's right, by "relax" it means that the new requirement is less strict that it used to be. This change is to support an application that is currently still on Node 18.12.

**Pull request merged**